docs/getting_started/quickstart.md

```diff
@@ -0,0 +1,58 @@
+# Quickstart
+
+This page explains how to quickly install and set up eZ Platform for development.
+
+## Basic requirements
+
+The recommended setup for using eZ Platform is:
```

Comment:
The recommended setup for using eZ Platform is described on the [requirements page](requirements_and_system_configuration.md).

Let's single source requirements page as it will be hard to track and update setup on many pages.

Reply:
Beg to differ :) I'd like this page to be something that a user can get started with without looking at other parts of the doc.
Although I do have doubts about listing the php extensions...